google-cloud-speech/acceptance/speech/async_test.rb

```diff
@@ -166,4 +166,21 @@
     results.first.confidence.must_be_close_to 0.9, 0.1
     results.first.alternatives.must_be :empty?
   end
+
+  it "recognizes audio with words" do
+    op = speech.process filepath, encoding: :linear16, language: "en-US", sample_rate: 16000, words: true
```

Comment:
We are migrating all speech client libraries to auto-generated client libraries. To minimize the impact on developers when we make that transition, we as service authors would strongly prefer that the manually generated libraries should use names consistent with what the auto-generated libraries will use for this feature; i.e. enable_word_time_offsets.

@lukesneeringer FYI.